src/components/HBox/HBox.tsx

```diff
@@ -1,22 +1,37 @@
 import * as React from 'react';
-import {HBox as CoreHBox, HBoxProps as CoreHBoxProps} from './coreHBox';
-import {ThemedComponent} from 'wix-ui-theme';
-import {theme} from './theme';
-import {Alignment} from './constants';
+import {number, bool, any, oneOf, arrayOf} from 'prop-types';
+import style from './HBox.st.css';
+import {addSpacing} from './utils';
 
-export interface HBoxProps extends CoreHBoxProps {
-  spacing?: string;
+export interface HBoxProps {
```

Comment:
Add documentation comments

Reply:
@hepiyellow I'm just moving the component over without changing anything about it. We can create a new task for documentation and anything else that needs to be improved.